examples/gnrc_networking/Makefile

```diff
@@ -29,6 +29,8 @@ USEMODULE += gnrc_icmpv6_echo
 USEMODULE += shell
 USEMODULE += shell_commands
 USEMODULE += ps
+USEMODULE += netstats_l2
+USEMODULE += netstats_ipv6
```

Comment:
What about compiling in `netstats_ipv6` automatically if `netstats` and `ipv6` are compiled?